contrib/terraform/aws/ssh-to-bastion.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+#
+# I am frequently connecting to the servers in the cluster and I tweak settings to
+# improve security or try to understand the innards of kubernetes. Because of this,
+# I wanted a script to hid the complexities of the SSH command. 
```

Comment:
What is this comment on the script?

Reply:
That comment was just a personal note. I agree that these scripts are for a specific inventory path. However, that path is what the `contrib/terraform/aws` procedure creates and these scripts  are in that directory. I feel the script is appropriate for anyone using that `aws` directory.